src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -0,0 +1,437 @@
+| SinglesRate                          | RECOMMENDED | [array][] of [numbers][] | Singles rate for each frame.                                                                                 |
+
+
+#### Example (`*_pet.json`)
+
+```JSON
+{
+	"Manufacturer": "Siemens",
+	"ManufacturersModelName": "High-Resolution Research Tomograph (HRRT, CTI/Siemens)",
+	"BodyPart": "Brain",
+	"BodyWeight": 21,
+	"BodyWeightUnit": "kg",
+	"Unit": "Bq/mL",
+	"TracerName": "CIMBI-36",
+	"TracerRadionuclide": "C11",
+	"TracerMolecularWeight": 380.28,
+	"TracerMolecularWeightUnit": "g/mol",
+	"TracerInjectionType": "bolus",
+	"InjectedRadioactivity": 573,
+	"InjectedRadioActivityUnit": "MBq",
+	"InjectedMass": 0.62,
+	"InjectedMassUnit": "ug",
+	"SpecificRadioactivity": 929.6,
+	"SpecificRadioactivityUnit": "MBq/ug",
+	"ModeOfAdministration": "bolus",
+	"MolarActivity": 353.51,
+	"MolarActivityUnit": "GBq/umol",
+	"MolarActivityMeasTime": "13:04:42",
+	"TimeZero": "13:04:42",
+	"ScanStart": 0,
+	"InjectionStart": 0,
+	"FrameTimesStart": [0, 10, 20, 30, 40, 50, 60, 80, 100, 120, 140, 160, 180, 240, 300, 360, 420, 480, 540, 660, 780, 900, 1020, 1140, 1260, 1380, 1500, 1800, 2100, 2400, 2700, 3000, 3300, 3600, 3900, 4200, 4500, 4800, 5100, 5400, 5700, 6000, 6300, 6600, 6900],
+	"FrameDuration": [10, 10, 10, 10, 10, 10, 20, 20, 20, 20, 20, 20, 60, 60, 60, 60, 60, 60, 120, 120, 120, 120, 120, 120, 120, 120, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300, 300],
+	"AcquisitionMode": "list mode",
+	"ImageDecayCorrected": true,
+	"ImageDecayCorrectionTime": 0,
+	"ReconMethodName": "3D-OSEM-PSF",
+	"ReconMethodParameterLabels": ["subsets","iterations"],
+	"ReconMethodParameterUnit": ["none","none"],
+	"ReconMethodParameterValues": [16,10],
+	"ReconFilterType": "none",
+	"AttenuationCorrection": "[137Cs]transmission scan-based"
+}
+```
+
+## Blood recording data
+
+This section includes all data needed to perform blood analysis for PET data.
+The section may be omitted completely if blood measurements of radioactivity were not made.
+
+Template:
+
+```Text
+sub-<participant_label>/
+ 	[ses-<session_label>/]
+	pet/
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>_blood.tsv
+	sub-<participant_label>[_ses-<session_label>][_task-<task_label>][_acq-<label>][_rec-<label>][_run-<index>]_recording-<label>_blood.json
+
+```
+
+Blood data belongs in the /pet folder along with the corresponding PET data.
+However, the blood data also follows the [inheritance principle](../02-common-principles.md#the-inheritance-principle) and may be moved to an upper level folder
+if it does not change, for example, with multiple reconstructions.
+The blood data is most often recorded using an autosampler for continuous blood samples,
```

Comment:
```suggestion
The blood data is often recorded using an autosampler for continuous blood samples,
```

I think it's more often than not recorded without an autosampler. This is clear for both.

Reply:
I agree with your suggested change.

Reply:
I agree as well.